webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -138,6 +137,89 @@ Kommunicate.attachmentEventHandler= {
         progressValue.style.strokeDashoffset = dashoffset;
         // value == 100 && !stopUpload && KommunicateUI.deleteProgressMeter(key);
         }      
+     },
+     handleSendingAttachment: function(e) {
+        var stopSending = !Kommunicate.internetStatus || $applozic(e.target).closest('.km-msg-box-attachment').find('.km-attachment-cancel-icon').hasClass('vis');
+        var upload= $applozic(e.target).closest('.km-msg-box-attachment').find('.km-attachment-upload-icon').hasClass('vis');
+        var attachmentDiv= $applozic(e.target).closest('.km-msg-box-attachment').children();
+        var msgKey = attachmentDiv[0].dataset.msgkey;
+        var deliveryStatusDiv= $applozic(e.target).closest('.mck-clear').find('.mck-msg-right-muted');
+        var file = KM_PENDING_ATTACHMENT_FILE[msgKey];
+        var fileMetaKey = attachmentDiv[0].dataset.filemetakey;
+        
+        if(stopSending) {
+            KommunicateUI.updateAttachmentStopUploadStatus(msgKey, true);
+            $applozic(".km-attachment-cancel-icon-"+msgKey).removeClass("vis").addClass("n-vis");
```

Comment:
@jithinbalu We can use the below function to add remove classes.
`kommunicateCommons.modifyClassList( {id : ["mck-rate-conversation"]}, "","n-vis");`

Reply:
@ab14bhardwaj existing code replaced with modifyClassList method